src/Symfony/Bundle/TwigBundle/ContainerAwareRuntimeLoader.php

```diff
@@ -11,8 +11,8 @@
 
 namespace Symfony\Bundle\TwigBundle;
 
+use Psr\Container\ContainerInterface;
 use Psr\Log\LoggerInterface;
```

Comment:
@fabpot as this class now depends only on PSR classes, should we add it in Twig itself ?